### ISSUE TITLE
Front page top section - use prime space to promote prime content

### ISSUE THREAD

**Issue by irinaz** [backdrop-ops/backdropcms.org#869]
I propose to replace image of mobile/desktop/phone of Backdrop front page with three sections linked to major sections of the site 

- Try backdrop
- Attend Backdrop Live
- Showcase of sites built with backdrop
<img width="1016" alt="Screen Shot 2022-02-10 at 12 59 08 PM" src="https://user-images.githubusercontent.com/2497005/153495702-387b1951-e798-4982-9680-e9d7f68e4b61.png">

**Comment by stpaultim** [comment]
@irinaz - I think that there are two issues here.

1) That we need to put a clear and prominent message on Backdrdopcms.org about Backdrop LIVE. 
2) That we should make better use of the space currently being use for the first block.

I agree with both of these suggestions, but I think we should deal with them separately. Yes, let's get a message near the top of the page promoting Backdrop LIVE ASAP. 

But, I am afraid that the discussion about a complete change to that block might need more discussion. Design changes to BackdropCMS.org do not usually happen quickly. By tying these two issues together, I'm afraid that we might not get our message about Backdrop LIVE up. 

NOTE: For the last event, I hacked a message about Backdrop LIVE onto the front page of BackdropCMS.org that wasn't really very pretty. I'd be happy to have some input on how to do that. 

I'm also happy to see something new done with this section of the site. I'm just not sure how quickly we can make that decision, but I am not opposed to making it quickly if that is possible. 

![image](https://user-images.githubusercontent.com/3144571/153665311-36227035-1e81-4c5f-bb93-c80a53e25aad.png)

**Comment by yorkshire-pudding** [comment]
There's also a question of tagline.  IMHO, that is too long to be a tag line.  The official one is "Put your content center stage" (I'll put up with the incorrect US spelling ;-) ) though there were a few others in the forum that sounded good too.  That is another discussion that may not be quick

**Comment by jenlampton** [comment]
> There's also a question of tagline.

yeah, that's not a tag line. That's an answer to the question "What is backdrop CMS?"

**Comment by jenlampton** [comment]
> I propose to replace image of mobile/desktop/phone of Backdrop front page with three sections linked to major sections of the site 

I love this idea, but the recommended screenshot needs some design love. I will reach out to our b.org designers and see if anyone has time time for a quick homepage update :) 

> and update tagline to "Modern lightweight CMS"

Backdrop is only "lightweight" in comparison to Drupal. Compared to any other CMS it's a beast, so it will give the wrong expectations to the larger web development community if we used this word. But, see also my comment above about this not being a tagline :)

**Comment by irinaz** [comment]
+1 "Put your content center stage"

**Comment by klonos** [comment]
> Backdrop is only "lightweight" in comparison to Drupal. Compared to any other CMS it's a beast...

I agree with that ^^

**Comment by jenlampton** [comment]
I spoke with @nicelobster last week and was able to get a fantastic solution for the top of our homepage: 

<img width="1290" alt="Screen Shot 2022-02-24 at 12 43 24 PM" src="https://user-images.githubusercontent.com/397895/155606301-f9eb95a9-d313-4792-9356-dde74653f703.png">

**Comment by yorkshire-pudding** [comment]
Looks great. Two minor points:
Should we expand CMS? For non technical users, CMS could be considered jargon.
Can we use the tagline "put your content centre stage", perhaps under the logo?

**Comment by jenlampton** [comment]
Great feedback!  I'm going to move your thoughts over to the [brand refresh issue](https://github.com/backdrop-ops/backdropcms.org/issues/874) because I want to get the block/hero style update out before Backdrop live.

**Comment by jenlampton** [comment]
I deployed my changes for phase I of the update: https://backdropcms.org. The showcase block isn't up there yet, so for now I put the old tagline into a custom block in its place.

**Comment by irinaz** [comment]
@jenlampton , I love it!  It is a real gift, thank you VERY much!

**Comment by yorkshire-pudding** [comment]
Looks great @jenlampton

**Comment by bugfolder** [comment]
Sorry, just found this issue now. I've got a couple of suggestions (which i made in the Zulip chat). Here's the current state:

![before](https://user-images.githubusercontent.com/29214084/156640876-d70dceef-899c-4b73-9845-0ea409e87043.jpg)

I found it a bit odd that the all-caps headings are at the top of the card for two of them but at the bottom on the "Featured Site" card. Might I suggest that middle card be something more like this?

![after](https://user-images.githubusercontent.com/29214084/156640768-f5fd155a-b850-47ec-adfd-a4c4921ae2ee.jpg)

So that reading across the headings, it's clear what's in each card.
And then maybe swap the first and third card, so that the first thing the visitor sees is "What is Backdrop CMS?" (at least in the LTR world), then the featured site, then BDLive. ?

**Comment by bugfolder** [comment]
> do you want me to ask for an explanation as to why?

Sure. 

Putting headings above, rather than the below, the material that they are describing, seem like a pretty standard design practice, and it makes it easy for the viewer to just scan across the cards to see what each card is about. When I first saw this, I wondered "why is this stuff on the BD site?" Then when I got to the bottom I see "oh yes, it's a featured site."

Also, I don't think we need an extra link "Take me Outside NSW" that duplicates the text we see immediately above. It's not part of the featured site, and in fact it's not clear visually where "featured site" stuff ends and "our site" stuff begins. (Hence my addition of faint gray rule lines for delineation.) I think most people would recognize that if we're putting up a screen shot of a featured site, it will be clickable and we wouldn't need an extra link.

**Comment by jenlampton** [comment]
> Also, I don't think we need an extra link "Take me Outside NSW" that duplicates the text we see immediately above. 

There is no text immediately above. The only text in that block is the heading, and the name of the site. I think you are confusing the screenshot with the text, maybe? Not all websites will have text that appears in their screenshot.

**Comment by jenlampton** [comment]
Maybe, we should choose a different featured site then. But first let me make the change to the image and let's go from there!

**Comment by BWPanda** [comment]
FTR, I agree that the block title above the screenshot would be good for consistency and also for less confusion about what the site is.

But I do **not** agree with removing the textual name/link if the site. As @jenlampton said, some screenshots won't have the name visible/readable, so knowing the name of the site/content (since this links to a node) is good.

**Comment by jenlampton** [comment]
If we use the hero image instead of the screenshot (which, I was supposed to do originally -- sorry about that folks) I think removing the site name becomes a non-issue: 

<img width="1296" alt="Screen Shot 2022-03-03 at 12 34 06 PM" src="https://user-images.githubusercontent.com/397895/156648400-4d187ef7-7209-44c4-b110-6adf90c61f4b.png">

I've got an email out to Nica asking about the titles and the block placements. In general, I believe that getting too many designers involved in any project tends to make the end result much worse than sticking with one vision, but if we can get the heading issue solved as part of her vision then everyone wins! 🤞

**Comment by bugfolder** [comment]
> I believe that getting too many designers involved in any project tends to make the end result much worse

I totally agree with this; in the end, it's the editor's vision that wins out (and I'm not the editor). But I do think it's OK to give feedback.

Using a hero image rather than a screenshot justifies the linkified title of the site. Although if we're going to feature a site, I think a screenshot of the site is a better fit for the goal of the card. (And not all sites will have a hero image.)

If we keep the linked-text name of the site in addition to a screen shot, consider making it smaller than the current bold, large type; I think it would serve its function at body-text-size, since the thing we want to draw people's attention to is the screen shot itself.

**Comment by jenlampton** [comment]
Just adding a note here that I have fixed my previous mistake with the screenshot (see below) and also sent an email to the designer requesting some of the other changes mentioned here :)

<img width="1365" alt="Screen Shot 2022-03-11 at 5 32 23 PM" src="https://user-images.githubusercontent.com/397895/157998385-28ef8310-9960-4566-b5c4-3c727637a345.png">
